### PR TITLE
Explicit batch and Einsum converter

### PR DESCRIPTION
**Pull request by jaybdub**

### CODE REVIEW

File: torch2trt/converters/cat.py [hunk 1]
```diff
@@ -16,7 +16,7 @@ def convert_cat(ctx):
     trt_inputs = broadcast_trt_tensors(ctx.network, trt_inputs, len(output.shape) - 1)
```
Comment (chaoz-dev) [inline torch2trt/converters/cat.py:16]: Just pattern matching, but do we need to remove the `- 1` here? ie. 
```
    trt_inputs = broadcast_trt_tensors(ctx.network, trt_inputs, len(output.shape))
```
I assume the dim reduction here was also due to the previous removal of the batch dim.

File: torch2trt/torch2trt.py [hunk 2]
```diff
@@ -534,6 +537,18 @@ def torch2trt(module,
...
+    
+    for i, name in enumerate(input_names):
+        # use fixed optimization profile
+        profile.set_shape(
+            name,
+            tuple(inputs[i].shape),
+            tuple(inputs[i].shape),
+            tuple(inputs[i].shape)
+        )
+        config.add_optimization_profile(profile)
```
Comment (chaoz-dev) [inline torch2trt/torch2trt.py:550]: Looks like this is for dynamic shapes? Is this necessary at the moment? I believe the current API will only support one (static) shape anyway, so I think we don't need _any_ optimization profiles (ie. we'll end up just using the given shape as before).

**Comment by SrivastavaKshitij** [comment]
Hey @jaybdub : I checked this PR with different Pytorch NGC Containers and it is working fine for standard models such as mobilenets, resnets, efficientnets with different versions of pytorch and tensorrt. 

In my case, I didnt have dynamic shapes, however, we need to resolve the issue @chaoz-dev mentioned above. 

I think explicit batch is a great feature and should be landed soon 

thanks

**Comment by jaybdub** [comment]
Thanks for the feedback on this PR @chaoz-dev  and @SrivastavaKshitij .

I've tested that the modification still pass all test cases.  Please let me know if you see any other issues, otherwise I'll merge shortly.

Thanks,
John

**Pull request merged**